site_script.js

```diff
@@ -111,7 +111,7 @@
     $body.classList.toggle('search__active', matchedIcons.length < icons.length);
 
     if (query === '') {
-      if ($orderByRelevance.classList.contains('active')) {
+      if ($body.classList.contains('order-by-relevance')) {
```

Comment:
Relatedly, [could you remove the `active` class here](https://github.com/simple-icons/simple-icons/blob/0b54463efd721c008b387f1b2bfabb8537280cd9/index.html#L217) and add `class="order-by-color"` to the body? This bug causes none of the ordering buttons to be highlighted on the first load.

Reply:
@mondeja, it seems you didn't `class="order-by-color"` to the body?

Reply:
Right, it's done! :hammer_and_wrench: